core/src/test/java/org/apache/iceberg/TestSequenceNumberForV2Table.java

```diff
@@ -309,6 +309,8 @@ public void testExpirationInTransaction() {
     V2Assert.assertEquals("Snapshot sequence number should be 1", 1, snap1.sequenceNumber());
     V2Assert.assertEquals(
         "Last sequence number should be 1", 1, readMetadata().lastSequenceNumber());
+    V2Assert.assertEquals(
+        "Should be 1 manifest list file", 1, listManifestListFiles(table.location()).size());
```

Comment:
Nit: could we just call it "manifest list" (the file is a bit redundant)

so the assertion message would be "Should be 1 manifest list"